src/bsp/platform/stm32fx_nucleo_wifi/xi_bsp_hton.h

```diff
@@ -0,0 +1,51 @@
+#ifndef __XI_BSP_HTON_H__
```

Comment:
It is. It is a networking function converting endianness between network byte order and host byte order. The thing is the WIFI. The STM32 WIFI SDK completely hides networking. We don't have access to hton low level function. Thus cn-cbor is in trouble on STM32 WIFI. That is why the implementation is here :(